package.json

```diff
@@ -18,12 +18,12 @@
     "sentry"
   ],
   "dependencies": {
-    "@atomist/cli": "0.2.2-20180731183747",
-    "@atomist/automation-client": "0.19.0-20180731181755",
-    "@atomist/automation-client-ext-dashboard": "0.1.0-20180705210657",
-    "@atomist/automation-client-ext-logzio": "0.1.0-20180628224529",
-    "@atomist/automation-client-ext-raven": "0.1.0-20180613131132",
-    "@atomist/slack-messages": "^0.12.1"
+    "@atomist/cli": "^1.2.0",
```

Comment:
Why is the cli here?

Reply:
I just updated the dependencies. I guess it can go, but I didn't want to start removing.

Reply:
It used to be needed when the build helpers were in the CLI and not provided by automation-client.